utils/streams/src/main/java/io/apicurio/registry/utils/streams/diservice/DefaultGrpcChannelProvider.java

```diff
@@ -14,6 +14,8 @@
     public Channel apply(HostInfo hostInfo) {
         return ManagedChannelBuilder
             .forAddress(hostInfo.host(), hostInfo.port())
+            .enableRetry()
+            .maxRetryAttempts(5)
```

Comment:
Does this seem reasonable to leave in?  I think so.  Retries are only problematic for RPC calls that mutate data, which I don't think is the case for the Streams storage?  I think all write operations are handled via the kafka streams processing, and only read operations potentially go through grpc.  Correct?

Reply:
Could make this configurable if we decide to leave it in...

Reply:
You're right. We're using gRPC to fetch info from read-only stores. I'll try to make it configurable.